src/redis_cmd.cc

```diff
@@ -2432,6 +2433,129 @@ class CommandZInterStore : public CommandZUnionStore {
   }
 };
 
+class CommandSortintAdd : public Commander {
+ public:
+  CommandSortintAdd() : Commander("siadd", -3, true) {}
+
+  Status Parse(const std::vector<std::string> &args) override {
+    try {
+      for (unsigned i = 2; i < args.size(); i++) {
+        auto id = std::stoi(args[i]);
```

Comment:
`std::stoi()` would return int type but the uint64 wants, use `std::stoull` here